.github/workflows/main.yml

```diff
@@ -74,6 +74,7 @@ jobs:
           key: ${{ env.FILE_NAME }}
 
   run-e2e-tests:
+    if: github.actor != 'dependabot'
```

Comment:
Seems that we need to test anyway.

Reply:
These test (with access to secrets) doesn't work for dependabot prs due to security reasons. You can check they are always red.

I usually combine dependabot PRs to one (we have action for it) and run CI for combined one manually.